app/assets/javascripts/student_profile/record_service.js

```diff
@@ -196,8 +197,7 @@
           style: styles.cancelRecordServiceButton,
           onClick: this.onClickCancel
         }, 'Cancel'),
-        (this.props.requestState === 'pending') ? dom.span({}, 'Saving...') : null,
-        (this.props.requestState === 'error') ? dom.span({}, 'Try again!') : null
+        (this.props.requestState === 'pending') ? dom.span({}, 'Saving...') : this.props.requestState
```

Comment:
What are the values that we expect `this.props.requestState` to have here? Do we want to show them to the user like this? (Legit asking).